com/ArchiveService.py

```diff
@@ -281,210 +315,287 @@ def verify_rinex_multiday(cnn, rinexinfo, Config):
         rnxlist.append(rnx.rinex)
         # some other file, move it to the repository
         retry_folder = os.path.join(Config.repository_data_in_retry,
-                                    'multidays_found/' + rnx.date.yyyy() + '/' + rnx.date.ddd())
+                                    'multidays_found/' + rnx.date.yyyy() +
+                                    '/' + rnx.date.ddd())
         rnx.compress_local_copyto(retry_folder)
 
-    # if the file corresponding to this session is found, assign its object to rinexinfo
-    event = pyEvents.Event(Description='%s was a multi-day rinex file. The following rinex files where generated '
-                                       'and moved to the repository/data_in_retry: %s. The file %s did not enter '
-                                       'the database at this time.' %
-                                       (rinexinfo.origin_file, ','.join(rnxlist), rinexinfo.crinez),
-                           NetworkCode = rinexinfo.NetworkCode,
-                           StationCode = rinexinfo.StationCode,
-                           Year        = int(rinexinfo.date.year),
-                           DOY         = int(rinexinfo.date.doy))
+    # if the file corresponding to this session is found,
+    #  assign its object to rinexinfo
+    event = pyEvents.Event(
```

Comment:
this style of continuation is appropriate here and more readable then our default alignment strategy that I mentioned earlier on line 219.  Thanks for cleaning it up!